bindings/python/benchmarks/benchmarks.py

```diff
@@ -53,16 +54,25 @@
 small.insert_many(
     [collections.OrderedDict([("x", 1), ("y", math.pi)]) for _ in range(N_SMALL_DOCS)]
 )
+base_small = collections.OrderedDict(
+    [("x", 1), ("y", math.pi), ("list", [math.pi for _ in range(64)])]
+)
+small.insert_many([{**base_small} for _ in range(N_SMALL_DOCS)])
 
 large_doc_keys = [c * i for c in string.ascii_lowercase for i in range(1, 101)]
 schemas[SMALL] = Schema({"x": pyarrow.int64(), "y": pyarrow.float64()})
+schemas[LIST + SMALL] = Schema({"list": pyarrow.list_(pyarrow.float64())})
 dtypes[SMALL] = np.dtype([("x", np.int64), ("y", np.float64)])
 dtypes[LARGE] = np.dtype([(k, np.float64) for k in large_doc_keys])
 schemas[LARGE] = Schema({k: pyarrow.float64() for k in large_doc_keys})
+schemas[LIST + LARGE] = Schema(
+    {k + "_list": pyarrow.list_(pyarrow.float64()) for k in large_doc_keys}
+)
 large = db[collection_names[LARGE]]
 large.drop()
 # 2600 keys: 'a', 'aa', 'aaa', .., 'zz..z'
-large_doc = collections.OrderedDict([(k, math.pi) for k in large_doc_keys])
```

Comment:
> # 2600 keys: 'a', 'aa', 'aaa', .., 'zz..z'

This is weird, I think we want to test a large number of keys, not a large number of really large key names. Large keys names is probably very uncommon. Can we change this to something more practical? In person you suggested `a1, a2, ..., a2600` which sounds good to me.

Reply:
Fixed.